### ISSUE TITLE
UHDBits search error

### ISSUE THREAD

**Issue by lattedesu** [Jackett/Jackett#888]
Hello, 

I get this inconsistent search results in UHDBits. 

```
Dec 25, 2016 07:28:03 PM	Info	
Manual search for "Snowden 2016" on UHDBits with 5 results.
Dec 25, 2016 07:27:18 PM	Info	
Found 0 torrentpotato releases from UHDBits for: Snowden 2016
```

Torrentpotato couldn't find the release, however manual search worked as you can see from the log.

I'm using with CouchPotato.

Can you check please? What am I doing wrong ?

This is my jackett version: Jackett Version 0.7.826.0 . Running with mono on Debian Jessie.

Thanks in advance,

**Comment by kaso17** [comment]
It's due to the very strict title filter in the potato provider, I'll make it more flexible so it works better.

**Comment by kaso17** [comment]
fixed with 657bb15282d1ee4f876b225ce847397646546c3b
